CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+# dbt_shopify v0.14.0
+
+[PR #92](https://github.com/fivetran/dbt_shopify/pull/92) includes the following updates:
+## Under the Hood
+- Adds enable/disable config for the `metadata` staging model using the `shopify_using_metafield` variable (default `true`).
+- Adds enable/disable config for the `abandoned_checkout` staging models using the `shopify_using_abandoned_checkout` variable (default `true`):
+   - `stg_shopify__abandoned_checkout`
+   - `stg_shopify__abandoned_checkout_discount_code`
+   - `stg_shopify__abandoned_checkout_shipping_line`. 
+- For more information on how to enable/disable these tables, refer to the [README](https://github.com/fivetran/dbt_shopify/blob/main/README.md#step-4-disable-models-for-non-existent-sources).
+- Updates the `index` calculation in `stg_shopify__abandoned_checkout_discount_code` by removing the conditional logic for null scenarios now that a disable config has been added to the model.
```

Comment:
This is not relevant for the transform model. We can remove this entry.
```suggestion
```

Reply:
Right! Removed